docs/development.rst

```diff
@@ -0,0 +1,160 @@
+=====================
+Development Standards
+=====================
+
+Front End Development
+=====================
+
+Background
+----------
+
+.. info::
+
+    Consider this the cononical resource for contributing Javascript and CSS. We
+    are currently in the process of modernizing our front end development
+    procedures. You will see a lot of different styles around the code base for
+    front end JavaScript and CSS.
+
+Our modern front end development stack includes the following tools:
+
+* `Gulp`_
+* `Bower`_
+* `Browserify`_
+* `Debowerify`_
```

Comment:
I'm using webpack + npm in a personal project and am really enjoying it. It also compiles SASS and NPM contained all the SASS dependencies I've been using so far so it works great. But that really boils down to a matter of taste. So I'm happy with browserify/debowerify/bower if that suits our need.

I just wondered because I thought browserify was made to use NPM as a package manager for the frontend :)